From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Krzysztof Halwa <krzysztof.halwa@solarwinds.com>
Date: Fri, 18 Dec 2020 13:46:55 +0100
Subject: Changed OutOfBlink cors flag to false.

Changed due to intercepted requests not falling into CORS path of making request.

diff --git a/third_party/blink/renderer/core/loader/threadable_loader.cc b/third_party/blink/renderer/core/loader/threadable_loader.cc
index 0fe36e76bd7e10d920b0ae6c8210adc8ad33c83a..3ae0e04f06b8d706cc59ef512db2175aa1846f95 100644
--- a/third_party/blink/renderer/core/loader/threadable_loader.cc
+++ b/third_party/blink/renderer/core/loader/threadable_loader.cc
@@ -208,7 +208,7 @@ ThreadableLoader::ThreadableLoader(
       execution_context_(execution_context),
       resource_fetcher_(resource_fetcher),
       resource_loader_options_(resource_loader_options),
-      out_of_blink_cors_(true),
+      out_of_blink_cors_(false),
       async_(resource_loader_options.synchronous_policy ==
              kRequestAsynchronously),
       request_context_(mojom::RequestContextType::UNSPECIFIED),
